lib/cosmos/system/target.rb

```diff
@@ -124,7 +124,9 @@ def process_file(filename)
           usage = "#{keyword} <FILENAME>"
           parser.verify_num_parameters(1, 1, usage)
           begin
-            Cosmos.require_file(parameters[0])
+            # Require absolute path to file in target lib folder.  Prevents name
+            # conflicts at the require step
+            Cosmos.require_file(File.join(@dir, 'lib', parameters[0]))
```

Comment:
I have a current COSMOS configuration with a Processor in my base lib folder. How do I now require that in my target and use it? This new code forces it to look in the target lib folder.